interface/main/calendar/add_edit_event.php

```diff
@@ -81,7 +87,7 @@
 if (isset($_GET['starttimeh'])) {
     $starttimeh = $_GET['starttimeh'];
     if (isset($_GET['startampm'])) {
-        if ($_GET['startampm'] == '2' && $starttimeh < 12) {
+        if ($_GET['startampm'] == '2' && $starttimeh < 12 && $globle_timeformat == 1) {
```

Comment:
Wouldn't it be simpler to just use $GLOBALS['time_display_format'] here and drop the code block at the top?

Reply:
Start time hour is set at the top, so I changed it at the bottom for simplicity.